Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.Networking.sb.in

```diff
@@ -73,11 +75,13 @@
     (require-all
         (socket-option-level SOL_SOCKET)
         (socket-option-name
+            SO_MARK_KNOWN_TRACKER
             SO_NECP_LISTENUUID
             SO_NOSIGPIPE
             SO_RCVBUF
             SO_REUSEADDR
-            SO_REUSEPORT)))
+            SO_REUSEPORT
+            SO_SNDLOWAT)))
```

Comment:
I am not able to see SO_SNDLOWAT being used on iOS from telemetry. Should we wait adding it until we see it in telemetry?

Reply:
That is reasonable. I'll remove that for iOS.